package.json

```diff
@@ -190,6 +190,7 @@
     "eslint-plugin-formatjs": "^2.18.0",
     "eslint-plugin-graphql": "^4.0.0",
     "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-jsx": "^0.1.0",
```

Comment:
It doesn't seem like this package is widely used: 3 stars on Github, ~10.000 weekly NPM downloads.
- What are other projects doing React + ESlint doing? Why are they not using it?
- I, among other team members, also use VSCode (Fedora), why am I not seeing this bug?

https://github.com/yannickcr/eslint-plugin-react seems more suited as a plugin and it's already imported in https://github.com/opencollective/eslint-config-opencollective/blob/main/index.js.